AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -30,6 +30,7 @@ public class FullscreenMessage: NSObject, WKNavigationDelegate, FullscreenPresen
     var listener: FullscreenMessageDelegate?
```

Comment:
Not related to your changes but for memory sake, we need to make this `weak` to avoid a retain cycle. Let's also rename it to `delegate`.

`weak var delegate: FullscreenMessageDelegate?`

https://medium.com/macoclock/delegate-retain-cycle-in-swift-4d9c813d0544

Reply:
Decided to stick with the `listener`  as opposed to  Apple friendly `delegate`... since  the publicAPI `CreateFullScreenMessage`   already calls this variable  `listener`... Switching only the internal variable name to  delegate might end up being confusing